### ISSUE TITLE
Bad example in Crate docs, `from_file` not implemented in ManifestStore

### ISSUE THREAD

**Issue by ronald-das** [contentauth/c2pa-rs#92]
Hi, while going through the example in https://docs.rs/c2pa/latest/c2pa/#example-reading-a-manifeststore  
Am getting not implemented error (E0599)
```
error[E0599]: no function or associated item named `from_file` found for struct `ManifestStore` in the current scope
 --> src/main.rs:4:41
  |
4 |     let manifest_store = ManifestStore::from_file("tests/fixtures/C.jpg")?;
  |                                         ^^^^^^^^^ function or associated item not found in `ManifestStore`

```
My cargo.toml file
```
[dependencies]
c2pa = "0.11.0"
```

**Comment by ronald-das** [comment]
After going through the struct documentation https://docs.rs/c2pa/latest/c2pa/struct.ManifestStore.html#method.from_file , `file_io` feature needs to enabled for it to work.

New Cargo.toml file
```
[dependencies]
c2pa = {version="0.11.0", features=["file_io"]}
```

**Comment by scouten-adobe** [comment]
@ronald-das thanks for bringing this to our attention. I'm reopening the ticket. I think we should add a note in the README to mention that `file_io` feature is needed. @crandmck or @gpeacock, would one of you please add this?

**Comment by crandmck** [comment]
Should we just change the `Cargo.toml` file in "Usage" section to be 
```
c2pa = {version="0.11.0", features=["file_io"]}
```
or should we qualify it by just adding something like "If you want to read or write a manifest file, add this to your `Cargo.toml`..." ?

**Comment by crandmck** [comment]
Addressed (partially, at least) by #93.
